pages/common/tlmgr-recreate-tlpdb.md

```diff
@@ -0,0 +1,9 @@
+# tlmgr recreate-tlpdb
+
+> Recreate the TeX Live package database.
+> This command should be used with caution, because it is poorly documented.
```

Comment:
Not completely sure this line should be here.

Reply:
Well, the command is clearly not meant to be used by end-users. I assume it's for developers only, which means there is no guarantee that it is safe to use in all cases, especially since you have to run it as root.
I'd say we either warn the user or we don't document the command at all.

Reply:
Ok, I think the current wording is a little too negative, how do you feel about

```suggestion
> This command has a lack of documentation and should be used with caution. 
```

Reply:
Both sounds good, @marchersimon which do you like more?